integration/loader-test.ts

```diff
@@ -69,27 +69,48 @@ test.describe("loader in an app", () => {
   let appFixture: AppFixture;
 
   let HOME_PAGE_TEXT = "hello world";
+  let REDIRECT_TARGET_TEXT = "redirect target";
+  let FETCH_TARGET_TEXT = "fetch target";
 
   test.beforeAll(async () => {
     appFixture = await createAppFixture(
       await createFixture({
         files: {
           "app/root.jsx": js`
+            import { Outlet } from '@remix-run/react'
+
             export default function Root() {
               return (
                 <html>
                   <body>
                     ${HOME_PAGE_TEXT}
+                    <Outlet />
                   </body>
                 </html>
               );
             }
           `,
           "app/routes/redirect.jsx": js`
             import { redirect } from "@remix-run/node";
-            export const loader = () => redirect("/");
+            export const loader = () => redirect("/redirect-target");
             export default () => <div>Yo</div>
           `,
+          "app/routes/redirect-target.jsx": js`
+            export default () => <div>${REDIRECT_TARGET_TEXT}</div>
+          `,
+          "app/routes/fetch.jsx": js`
+            export function loader({ request }) {
+              return fetch(new URL(request.url).origin + '/fetch-target');
```

Comment:
This is the case we're fixing here - returning a raw `fetch` `Response` wasn't matching our `instanceof Response` check